aneris/cli.py

```diff
@@ -63,25 +64,28 @@ def harmonize(inf, history, regions, rc, output_path, output_prefix, return_resu
     for scenario in driver.scenarios():
         driver.harmonize(scenario)
     model, metadata, diagnostics = driver.harmonized_results()
+
     if return_result:
         return model, metadata, diagnostics
 
-    # write to excel
-    prefix = output_prefix or inf.split('.')[0]
-    fname = os.path.join(output_path, '{}_harmonized.xlsx'.format(prefix))
-    logger().info('Writing result to: {}'.format(fname))
-    aneris.pd_write(model, fname, sheet_name='data')
-
-    # save data about harmonization
-    fname = os.path.join(output_path, '{}_metadata.xlsx'.format(prefix))
-    logger().info('Writing metadata to: {}'.format(fname))
-    aneris.pd_write(metadata, fname)
-
-    # save data about harmonization
-    if not diagnostics.empty:
-        fname = os.path.join(output_path, '{}_diagnostics.xlsx'.format(prefix))
-        logger().info('Writing diagnostics to: {}'.format(fname))
-        aneris.pd_write(diagnostics, fname)
+    if write_output:
```

Comment:
out of curiosity - why is `return_result` not sufficient? if you use that option, then the workflow stops on return?

Reply:
Ah my intention was to make the option for writing out the results more explicit and allow for a combination of both options, but now I see I should have changed the order of the two if-statements to make this work.

Reply:
Cool, thanks!